client/src/components/InsertShorty.js

```diff
@@ -1,6 +1,19 @@
 import React, { useState } from "react";
 import PropTypes from "prop-types";
 import { postNewShorty } from "../api/shorties";
+import styled from "styled-components/macro";
+
+const Form = styled.form`
```

Comment:
Thx, but I don't agree with this code style. 
I prefer to first define functions/variables before I use them. I think it is very confusing if some components are imported in the first lines and others are defined at the end of the module.

If you have issues finding the exported function, it might be a different issue. Maybe there are too many components defined (maybe code splitting?).

Reply:
Yes, I understand that, I like it to have my styles at the bottom of my components, but I think it's not important. Your point about function calls is really good, thank you!